cmd/chunks-inspect/labels.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"sort"
+	"strconv"
+)
+
+type Label struct {
```

Comment:
can you import prometheus/labels ? seems to contains all you need.

https://github.com/prometheus/prometheus/blob/master/pkg/labels/labels.go

It is already vendored.

Reply:
I wanted to keep dependencies at minimum (hence a separate module as well). But I guess it doesn't matter too much.